PROCESSES.md

```diff
@@ -0,0 +1,46 @@
+# Processes
+
+This document outlines processes and procedures for some common tasks in the charts repository.
+
+## Deprecating A Chart
+
+When a chart is no longer maintained it can be [deprecated](https://en.wikipedia.org/wiki/Deprecation). Once a chart is deprecated the expectation is the chart will see no further development. The chart and its versions will still be accessible, though tools such as [monocular](https://github.com/kubernetes-helm/monocular) and [Kubeapps Hub](https://hub.kubeapps.com/) will no longer list the chart.
+
+To deprecate a chart perform the following:
+
+1. Increment the chart `version` in the `Chart.yaml` file. This is required as all charts are immutable.
+1. Add a property to the `Chart.yaml` file of `deprecated: true` at the top level of the YAML structure.
+1. Above the deprecated property add a comment noting that the chart is deprecated and linking to the deprecation policy.
+1. Remove any maintainers from the chart as the chart is no longer maintained.
```

Comment:
We should also add:
- prefix description with "DEPRECATED"
- Update READMEs and NOTES.txt to note that the chart is deprecated

This is needed because Helm currently does not treat the deprecated field in any special way, so it won't be obvious to users of the CLI.

Reply:
@prydonius good call. I updated the PR to include those.